teku/src/main/java/tech/pegasys/teku/cli/slashingprotection/ExportCommand.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.cli.slashingprotection;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import org.apache.logging.log4j.util.Strings;
+import picocli.CommandLine;
+import tech.pegasys.teku.cli.options.DataOptions;
+import tech.pegasys.teku.data.SlashingProtectionExporter;
+import tech.pegasys.teku.logging.SubCommandLogger;
+import tech.pegasys.teku.util.cli.PicoCliVersionProvider;
+
+@CommandLine.Command(
+    name = "export",
+    description = "Export slashing protection database in Minimal format.",
+    mixinStandardHelpOptions = true,
+    abbreviateSynopsis = true,
+    versionProvider = PicoCliVersionProvider.class,
+    synopsisHeading = "%n",
+    descriptionHeading = "%nDescription:%n%n",
+    optionListHeading = "%nOptions:%n",
+    footerHeading = "%n",
+    footer = "Teku is licensed under the Apache License 2.0")
+public class ExportCommand implements Runnable {
+  public static final SubCommandLogger SUB_COMMAND_LOG = new SubCommandLogger();
+
+  @CommandLine.Mixin(name = "Data")
+  private DataOptions dataOptions;
+
+  @CommandLine.Option(
+      names = {"--to"},
+      paramLabel = "<FILENAME>",
+      description = "The file to export the slashing protection database to.",
+      required = true,
+      arity = "1")
+  private String toFileName = Strings.EMPTY;
+
+  @Override
+  public void run() {
+
+    final Path slashProtectionPath =
+        DataOptions.getValidatorsSlashingProtectionPath(dataOptions.getDataPath());
+    verifySlashingProtectionPathExists(slashProtectionPath);
+
+    SlashingProtectionExporter slashingProtectionExporter = new SlashingProtectionExporter();
+
+    SUB_COMMAND_LOG.display("Reading slashing protection data from: " + slashProtectionPath);
+    slashingProtectionExporter.initialise(slashProtectionPath);
```

Comment:
We should probably display an error an exit if there are no slashing protection records found (ie the directory exists but is empty)

Reply:
I could potentially also output each file that I load? I ended up doing this on import, but maybe i should do it on export also...